app/assets/javascripts/application.js

```diff
@@ -13,6 +13,8 @@
 //= require jquery_ujs
 //= require dsds
 //= require date_chooser
+//= require show_hide
+//= require disable_checker
```

Comment:
Benefit of having these in the application JS file is that the functionality is available for all views and controllers without further code.

A different approach would be to [put them in a controller specific coffeescript file](http://guides.rubyonrails.org/asset_pipeline.html#controller-specific-assets). However I get the sense these form partials may get rendered via more than one controller.